### PR TITLE
Adds scripts for four example R models, using plumber

### PR DESCRIPTION
**Pull request by swilliamson-cs**
Two binary classification examples, two regression examples.

The regression examples are based on the longley dataset in R. Inputs are 6-dimensional numeric. 

The classification examples are based on the BreastCancer dataset in mlbench. Inputs are 10-dimensional numeric.

### CODE REVIEW

**Comment by mattsanchez** [comment]
@swilliamson-cs This is very good, thanks for contributing.  @akumar-cs This is a good opportunity to get our s2i builder for R done so we can package these models the same way we handle python.

File: examples/r_examples/lasso_api.R [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# lasso_api.R
+
+# Global code; gets executed at plumb() time.
+if(!require(glmnet)){
+  install.packages("glmnet")
+}
+library(glmnet)
+data(longley)
```
Comment (sdraper-CS) [inline examples/r_examples/lasso_api.R]: @swilliamson-cs Where does this data actually come from (as in how is it loaded)?  If we want to be able to perform Certifai analysis we need it as explicit CSVs (also ideally the models would train from provided CSVs, but that point is perhaps more finessable).  Similarly the other datasets

Comment (swilliamson-cs) [reply examples/r_examples/lasso_api.R]: I've changed it to use German Credit and Auto Insurance as appropriate, and cleaned up the implementations a little.

**Review by cdraper-CS — APPROVED** [review]

**Comment by cdraper-CS** [comment]
Closing this PR. The R example will be added in cortex-certifai-examples repo.

**Pull request closed without merging**